strawberry/federation.py

```diff
@@ -115,8 +116,15 @@ def entities_resolver(self, root, info, representations):
             type = self.schema_converter.type_map[type_name]
 
             definition = cast(TypeDefinition, type.definition)
+            resolve_reference = definition.origin.resolve_reference
 
-            results.append(definition.origin.resolve_reference(**representation))
+            func_args = get_func_args(resolve_reference)
+            kwargs = representation
+
+            if "info" in func_args:
+                kwargs["info"] = info
+
+            results.append(resolve_reference(**kwargs))
```

Comment:
I think we'll change this in future to follow's @BryceBeagle structure, but for now this is good 😊